test/functional/utils/test-bytes.js

```diff
@@ -33,4 +46,22 @@ describe('stringToBytes', function() {
     }).to.throw();
   });
 
+  it('should convert bytes array to string', () => {
+    const string = bytesToString(new Uint8Array([102, 111, 111]));
+    expect(string).to.equal('foo');
+  });
+
+  it('should generate random bytes array', () => {
+    sandbox.stub(Math, 'random').returns(0.111111111111111111111111111111);
```

Comment:
no need to use sandbox.


`Math.random = () => 0.11111111111111111111111;` does the same job